src/include/daos_srv/bio.h

```diff
@@ -86,6 +86,28 @@ struct bio_blob_hdr {
 	uuid_t		bbh_pool;
 };
 
+/*
+ * Current device state (health statistics). Periodically updated in
+ * bio_bs_monitor(). Used to determine faulty device status.
+ */
+struct bio_device_health_state {
+	uint64_t	 bhs_timestamp;
+	int		 bhs_bio_read_errs;
+	int		 bhs_bio_write_errs;
+	int		 bhs_bio_unmap_errs;
```

Comment:
It's better to use uint32_t instead of int, and shift them around to make fields 64bits aligned.

Reply:
not sure I understand? what do you mean shift them around? I can make them uint32_t but then they should still be last in the struct for alignment, right?

Reply:
ok will make the change